### PR TITLE
Battle ship lib pull request

### PR DESCRIPTION
**Pull request by xKoem**
BattleSheepLib - library for battle sheep.
Models and 3 small services.
You can focus mostly on board package.

### CODE REVIEW

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java [hunk 1]
```diff
@@ -48,22 +50,44 @@ private Mast getMast(Coordinate coordinate) throws NoMastAtPositionException {
...
             return mastAtCoordinate.get();
         }
 
         throw new NoMastAtPositionException(coordinate);
     }
 
+    public List<Coordinate> getCoordinates() {
+        List<Coordinate> shipCoordinates = new ArrayList<>();
+
+        mastList.stream()
```
Comment (totine) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java]: collector in stream:
```
        List<Coordinate> coordinates = mastList.stream()
                .map(Mast::getCoordinate)
                .collect(Collectors.toList());

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java [hunk 2]
```diff
@@ -48,22 +50,44 @@ private Mast getMast(Coordinate coordinate) throws NoMastAtPositionException {
                 .filter(mast -> mast.getCoordinate().equals(coordinate))
                 .findFirst();
 
-        if (mastAtCoordinate.isPresent()){
+        if(mastAtCoordinate.isPresent()) {
```
Comment (totine) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java]: space after if

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipAmount.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package pl.jakubowskiprzemyslaw.tajgertim.models.configuration.ship;
+
+import java.io.Serializable;
+
+public class ShipAmount implements Serializable {
```
Comment (totine) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipAmount.java]: ovijkozen

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipConfiguration.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+    private Map<ShipSize, ShipAmount> shipConfiguration = new HashMap<>();
+
+    public void put(ShipSize shipSize, ShipAmount shipAmount) {
+        shipConfiguration.put(shipSize, shipAmount);
+    }
+
+    public int getBoardSize() {
+        int fleetSize = getFleetSize();
+        double minimalBoardSize = Math.sqrt(fleetSize / .2);
```
Comment (totine) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipConfiguration.java]: magic number

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipSize.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
+package pl.jakubowskiprzemyslaw.tajgertim.models.configuration.ship;
+
+public enum ShipSize {
+    mast1 {
```
Comment (totine) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipSize.java]: CAPITALS

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/coordinates/Coordinate.java [hunk 6]
```diff
@@ -14,6 +14,10 @@ public Coordinate(int x, int y) {
         this.y = y;
     }
 
+    public Coordinate moveCoordinate(Coordinate coordinate) {
```
Comment (totine) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/coordinates/Coordinate.java]: translate

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+        int x = random.nextInt(10);
+        int y = random.nextInt(10);
+        return new Coordinate(x, y);
+    }
+
+    @Test(invocationCount = 20)
+    public void markHit_checkIfIsHit() throws NoMastAtPositionException {
+        board.markHit(coordinate);
+
+        FieldState fieldState = board.getCoordinateStatus(coordinate);
```
Comment (makoJohn) [inline BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: notice how you wanted to get a coordinate STATUS, but got a field STATE

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/coordinates/Coordinate.java [hunk 8]
```diff
@@ -45,4 +48,7 @@ public int getY() {
         return y;
     }
 
+    public boolean isInRange(int range) {
```
Comment (totine) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/coordinates/Coordinate.java]: Is less than

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+                "..........\n" +
+                "..........\n";
+
+        String consoleView = board.getConsoleView(10);
+
+        assertEquals(consoleView, expectedView);
+
+    }
+
+}
```
Comment (totine) [inline BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: enter

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+        coordinate = getRandomCoordinate();
+        Mast mast = new Mast(coordinate);
+        ship.addMastToShip(mast);
+
+        fleet = new Fleet(Arrays.asList(ship));
+
+        board = new Board(fleet);
+    }
+
+    private Coordinate getRandomCoordinate() {
```
Comment (totine) [inline BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: consider data provider

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/FleetTest.java [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+        fleet.addShip(ship);
+        List<Ship> expectedShips = Arrays.asList(ship);
+
+        List<Ship> ships = fleet.getShips();
+
+        assertEquals(ships, expectedShips);
+    }
+
+
+}
```
Comment (totine) [inline BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/FleetTest.java]: enter

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+        Random random = new Random();
+        int x = random.nextInt(10);
+        int y = random.nextInt(10);
+        return new Coordinate(x, y);
+    }
+
+    @Test(invocationCount = 20)
+    public void markHit_checkIfIsHit() throws NoMastAtPositionException {
```
Comment (makoJohn) [inline BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: change the name of the method.

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java [hunk 13]
```diff
@@ -0,0 +1,90 @@
...
+    }
+
+    @Test(invocationCount = 20, expectedExceptions = NoMastAtPositionException.class)
+    public void markHitOnNotExistingField() throws NoMastAtPositionException {
+        Coordinate nextCoordinate = coordinate.moveCoordinate(new Coordinate(1, 0));
+        board.markHit(nextCoordinate);
+    }
+
+    @Test(invocationCount = 20)
+    public void getShipList() {
```
Comment (makoJohn) [inline BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: useless

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java [hunk 14]
```diff
@@ -0,0 +1,90 @@
...
+    public void markHit_checkIfIsHit() throws NoMastAtPositionException {
+        board.markHit(coordinate);
+
+        FieldState fieldState = board.getCoordinateStatus(coordinate);
+
+        Assert.assertEquals(fieldState, FieldState.HIT_MAST);
+    }
+
+    @Test(invocationCount = 20, expectedExceptions = NoMastAtPositionException.class)
+    public void markHitOnNotExistingField() throws NoMastAtPositionException {
```
Comment (makoJohn) [inline BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: why do you need 20 invocations of this?

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java [hunk 15]
```diff
@@ -0,0 +1,90 @@
...
+                ".*........\n" +
+                "..........\n" +
+                "..........\n" +
+                "..........\n" +
+                "..........\n" +
+                "..........\n" +
+                "..........\n" +
+                "..........\n";
+
+        String consoleView = board.getConsoleView(10);
```
Comment (makoJohn) [inline BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: what is 10?

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java [hunk 16]
```diff
@@ -0,0 +1,90 @@
...
+        assertEquals(shipList, expectedShipList);
+    }
+
+    @Test
+    public void checkConsoleView() {
+        Coordinate coordinate = new Coordinate(1,2);
+        Ship ship = new Ship();
+        ship.addMastToShip(new Mast(coordinate));
+        Fleet fleet = new Fleet(Arrays.asList(ship));
+        Board board = new Board(fleet);
```
Comment (makoJohn) [inline BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: arrangments are not clear.
You can create a static factory method to create a board.

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/NoMastAtPositionException.java [hunk 17]
```diff
@@ -5,7 +5,7 @@
 public class NoMastAtPositionException extends Throwable {
```
[the remark this answers is not in the document]
Comment (xKoem) [reply BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/NoMastAtPositionException.java]: done

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Mast.java [hunk 18]
```diff
@@ -37,7 +37,6 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
```
[the remark this answers is not in the document]
Comment (xKoem) [reply BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Mast.java]: we have to use only Coordinate

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java [hunk 19]
```diff
@@ -6,17 +6,20 @@
...
     private List<Mast> mastList;
 
-    public Ship(List<Mast> mastList) {
+    private Ship(List<Mast> mastList) {
         this.mastList = mastList;
     }
 
-    Ship() {
+    public Ship() {
+
```
[the remark this answers is not in the document]
Comment (xKoem) [reply BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java]: default constructor is necessary for now on

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java (hunk 1 above)
Comment (xKoem) [reply BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java]: done

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java (hunk 2 above)
Comment (xKoem) [reply BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java]: done

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipAmount.java (hunk 3 above)
Comment (xKoem) [reply BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipAmount.java]: yup

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipConfiguration.java (hunk 4 above)
Comment (xKoem) [reply BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipConfiguration.java]: done

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipSize.java (hunk 5 above)
Comment (xKoem) [reply BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipSize.java]: DONE

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/coordinates/Coordinate.java (hunk 6 above)
Comment (xKoem) [reply BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/coordinates/Coordinate.java]: done

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/coordinates/Coordinate.java (hunk 8 above)
Comment (xKoem) [reply BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/coordinates/Coordinate.java]: done

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java (hunk 10 above)
Comment (xKoem) [reply BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: maybe later

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/FleetTest.java (hunk 11 above)
Comment (xKoem) [reply BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/FleetTest.java]: done

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java (hunk 9 above)
Comment (xKoem) [reply BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: done

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Fleet.java [hunk 20]
```diff
@@ -25,12 +25,16 @@ public Fleet(Fleet fleet) {
         ships = new ArrayList<>(fleet.getShips());
     }
 
+
+    public void addShip(Ship ship) {
+        ships.add(ship);
+    }
+
     List<Ship> getShips() {
         return Collections.unmodifiableList(ships);
```
Comment (gacikjakub) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Fleet.java]: :+1:

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Mast.java [hunk 21]
```diff
@@ -19,11 +19,11 @@ public Coordinate getCoordinate() {
         return coordinate;
     }
 
-    public MastState getMastState() {
+    MastState getMastState() {
         return mastState;
     }
 
-    public void markAsHit() {
```
Comment (gacikjakub) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Mast.java]: :+1:

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java [hunk 22]
```diff
@@ -36,8 +39,7 @@ FieldState getMastFieldState(Coordinate coordinate) throws NoMastAtPositionExcep
         MastState mastState = getMast(coordinate).getMastState();
         if (mastState.equals(MastState.HIT)) {
```
Comment (gacikjakub) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java]: return mastState.equals(MastState.HIT) ? FieldState.HIT_MAST : FieldState.NOT_HIT_MAST;
 ????????? It's IMO only

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java [hunk 23]
```diff
@@ -48,22 +50,41 @@ private Mast getMast(Coordinate coordinate) throws NoMastAtPositionException {
...
 
-        if (mastAtCoordinate.isPresent()){
+        if (mastAtCoordinate.isPresent()) {
             return mastAtCoordinate.get();
         }
 
         throw new NoMastAtPositionException(coordinate);
     }
 
+    public List<Coordinate> getCoordinates() {
```
Comment (gacikjakub) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/Ship.java]: :+1:

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/FrontGameConfiguration.java [hunk 24]
```diff
@@ -0,0 +1,60 @@
+package pl.jakubowskiprzemyslaw.tajgertim.models.configuration;
+
+public class FrontGameConfiguration { //I don't like it, I love it, love it, love it, uh oh, but in reality I don't. Please suggest better solution.
```
Comment (gacikjakub) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/FrontGameConfiguration.java]: :joy:

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/FrontGameConfiguration.java [hunk 25]
```diff
@@ -0,0 +1,60 @@
...
+    public String toString() {
+        return "FrontGameConfiguration{" +
+                "gameName='" + gameName + '\'' +
+                ", MAST_1='" + mast1 + '\'' +
+                ", MAST_2='" + mast2 + '\'' +
+                ", MAST_3='" + mast3 + '\'' +
+                ", MAST_4='" + mast4 + '\'' +
+                '}';
+    }
+}
```
Comment (gacikjakub) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/FrontGameConfiguration.java]: :joy: :joy:  :joy:  :joy:  :joy:  :joy:  :joy:

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/GameConfiguration.java [hunk 26]
```diff
@@ -1,27 +1,52 @@
...
+        this.shipConfiguration = new ShipConfiguration();
     }
 
-    public void setGameName(String gameName) {
-        this.gameName = gameName;
+    public GameConfiguration(FrontGameConfiguration frontGameConfiguration) {
+        this.shipConfiguration = new ShipConfiguration();
+        this.gameName = frontGameConfiguration.getGameName();
+        this.shipConfiguration.put(ShipSize.MAST_1, new ShipAmount(frontGameConfiguration.getMast1()));
+        this.shipConfiguration.put(ShipSize.MAST_2, new ShipAmount(frontGameConfiguration.getMast2()));
```
Comment (gacikjakub) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/GameConfiguration.java]: :joy:

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipConfiguration.java [hunk 27]
```diff
@@ -0,0 +1,61 @@
...
+            shipPlacementList.addAll(creteShipPlacementListForSpecificShipSize(shipSize));
+        }
+
+        shipPlacementList.sort((o1, o2) -> o1 < o2? 1: -1);
+
+        return shipPlacementList;
+    }
+
+    int getFleetSize() {
+        return shipConfiguration.keySet()
```
Comment (gacikjakub) [inline BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipConfiguration.java]: :+1:

File: BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/FrontGameConfiguration.java (hunk 24 above)
Comment (xKoem) [reply BattleSheepLib/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/FrontGameConfiguration.java]: Tracked under #119

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/services/QueueServiceTest.java [hunk 28]
```diff
@@ -0,0 +1,27 @@
...
+
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pl.jakubowskiprzemyslaw.tajgertim.models.QueueObject;
+
+import static org.mockito.Mockito.mock;
+
+@Test
+public class QueueServiceTest {
```
[the remark this answers is not in the document]
Comment (boczkas) [reply BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/services/QueueServiceTest.java]: Test class removed.
It is hard to test it (new in constructor).
Class refactor planned.
Tracked under: #121

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipConfigurationTest.java [hunk 29]
```diff
@@ -0,0 +1,61 @@
...
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.testng.Assert.*;
+
+@Test
+public class ShipConfigurationTest {
+
+    private ShipConfiguration shipConfiguration;
```
[the remark this answers is not in the document]
Comment (boczkas) [reply BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/ShipConfigurationTest.java]: This specific configuration holds logic inside it (so maybe name is not the best), so tests in this area gives us really nice confidence level in used later placing algorithm.

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/CoordinateTest.java [hunk 30]
```diff
@@ -0,0 +1,25 @@
+package pl.jakubowskiprzemyslaw.tajgertim.models.configuration.ship;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pl.jakubowskiprzemyslaw.tajgertim.models.coordinates.Coordinate;
+
+@Test
+public class CoordinateTest {
```
[the remark this answers is not in the document]
Comment (boczkas) [reply BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/configuration/ship/CoordinateTest.java]: it occurred that this method is used only in tests. It has been removed

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java (hunk 7 above)
Comment (boczkas) [reply BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: Agree in 100%, but it needs deeper refactor.
Will be done later.
It is tracked under issue #129.

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java (hunk 12 above)
Comment (boczkas) [reply BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: done

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java (hunk 13 above)
Comment (boczkas) [reply BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: removed

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java (hunk 14 above)
Comment (boczkas) [reply BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: 1 invocation now

File: BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java (hunk 15 above)
Comment (boczkas) [reply BattleSheepLib/src/test/java/pl/jakubowskiprzemyslaw/tajgertim/models/board/BoardTest.java]: variable extracted

**Pull request merged**